### PR TITLE
Simplify scripts

### PR DESCRIPTION
**Pull request by carmenromo**
A few functions have been added to the scripts we use in ANTEA to simplify the code and avoid bugs by repeating the same lines. Also the script `reconstruct_phot.py` has been updated because it was using old ANTEA functions.

### CODE REVIEW

File: antea/scripts/reconstruct_phot.py [hunk 1]
```diff
@@ -98,21 +103,11 @@
...
         r1 = r2 = None
         if len(pos1) > 0:
-            pos1_phi = rf.from_cartesian_to_cyl(np.array(pos1))[:,1]
-            diff_sign = min(pos1_phi ) < 0 < max(pos1_phi)
-            if diff_sign & (np.abs(np.min(pos1_phi))>np.pi/2.):
-                pos1_phi[pos1_phi<0] = np.pi + np.pi + pos1_phi[pos1_phi<0]
-            mean_phi = np.average(pos1_phi, weights=q1)
-            var_phi1 = np.average((pos1_phi-mean_phi)**2, weights=q1)
-            r1  = Rpos(np.sqrt(var_phi1)).value
+            _, var_phi1 = rf.phi_mean_var(pos1, q1)
```
Comment (paolafer) [inline antea/scripts/reconstruct_phot.py:106]: Here the argument of `phi_mean_var` is a list of `phi` coordinates, not the cartesian 3D position.

File: antea/scripts/reconstruct_phot.py [hunk 2]
```diff
@@ -98,21 +103,11 @@
...
+            r1          = Rpos(np.sqrt(var_phi1)).value
         if len(pos2) > 0:
-            pos2_phi = rf.from_cartesian_to_cyl(np.array(pos2))[:,1]
-            diff_sign = min(pos2_phi ) < 0 < max(pos2_phi)
-            if diff_sign & (np.abs(np.min(pos2_phi))>np.pi/2.):
-                pos2_phi[pos2_phi<0] = np.pi + np.pi + pos2_phi[pos2_phi<0]
-            mean_phi = np.average(pos2_phi, weights=q2)
-            var_phi2 = np.average((pos2_phi-mean_phi)**2, weights=q2)
-            r2  = Rpos(np.sqrt(var_phi2)).value
+            _, var_phi2 = rf.phi_mean_var(pos2, q2)
```
Comment (paolafer) [inline antea/scripts/reconstruct_phot.py:109]: Same as above.

File: antea/scripts/reconstruct_phot.py (hunk 1 above)
Comment (carmenromo) [reply antea/scripts/reconstruct_phot.py:106]: Right!

**Review by paolafer — APPROVED** [review]
This PR simplifies the existing scripts, reducing code duplication. Approved!

**Pull request merged**